guides/common/modules/proc_creating-a-content-view.adoc

```diff
@@ -10,23 +10,31 @@ While you can stipulate whether you want to resolve any package dependencies on
 For more information, see xref:Resolving_Package_Dependencies_{context}[].
 
 .Procedure
-. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create New View*.
+. In the {ProjectWebUI}, navigate to *Content* > *Content Views* and click *Create Content View*.
 . In the *Name* field, enter a name for the view.
 {Project} automatically completes the *Label* field from the name you enter.
 . In the *Description* field, enter a description of the view.
+. In the *Type* field, choose a *Component content view* or a *Composite content view*.
 . Optional: if you want to solve dependencies automatically every time you publish this Content View, select the *Solve Dependencies* check box.
 Dependency solving slows the publishing time and might ignore any Content View filters you use.
 This can also cause errors when resolving dependencies for errata.
-. Click *Save* to create the Content View.
-. In the *Repository Selection* area, select the repositories that you want to add to your Content View, then click *Add Repositories*.
-. Click *Publish New Version* and in the *Description* field, enter information about the version to log changes.
-. Click *Save*.
-. Optional: to force metadata regeneration on Yum repositories, from the *Actions* list for your Content View versions, select *Regenerate Repository Metadata*.
+. Click *Create Content View*.
+
+.Component Content View Steps
```

Comment:
This should be treated as a separate procedure? Modularity means one procedure per .adoc

Reply:
A new procedure was created **modules/proc_component-content-view-steps.adoc** and added to **assembly-managing-content-views.adoc** between **proc_creating-a-content-view.adoc** and **proc_viewing-module-streams.adoc**